### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by daler**
Start using logging for easier debugging when we have problems in the future.

### CODE REVIEW

**Comment by bgruening** [comment]
Great work @daler! On both PRs!

File: bioconda_utils/cli.py [hunk 1]
```diff
@@ -92,26 +92,28 @@ def dag(repository, packages="*", format='gml', hide_singletons=False):
...
         return
 
-    d, n2r = utils.get_dag(utils.get_recipes(repository, packages), restrict=False)
+    d, n2r = utils.get_dag(utils.get_recipes(recipe_folder, packages), restrict=False)
 
     recipes = []
     for dep in dependencies:
         for i in d[dep]:
-            recipes.extend(n2r[i])
+            for r in n2r[i]:
```
Comment (bgruening) [inline bioconda_utils/cli.py]: @daler why this change?

File: bioconda_utils/cli.py [hunk 2]
```diff
@@ -31,41 +32,43 @@
...
           packages="*",
           testonly=False,
-          verbose=False,
           force=False,
-          docker=None):
+          docker=None,
+          loglevel="warning",
+          ):
 
+    logging.basicConfig(format="%(asctime)s:%(levelname)s:%(name)s:%(message)s", level=getattr(logging, loglevel.upper()))
```
Comment (bgruening) [inline bioconda_utils/cli.py]: I think this should be on top after the logging import.

**Comment by johanneskoester** [comment]
Very nice. Thank you!
